pkg/curatedpackages/packagecontrollerclient.go

```diff
@@ -127,9 +127,6 @@ func (pc *PackageControllerClient) CreateCredentials(ctx context.Context) error
 		return errors.New("environment variables EKSA_AWS_SECRET_ACCESS_KEY and EKSA_AWS_ACCESS_KEY_ID not provided")
 	}
 
```

Comment:
Why is this being deleted?

Reply:
As per my discussion with @junshun , this is not necessary anymore.